tests/check-http-method.spec.js

```diff
@@ -0,0 +1,20 @@
+const HTTP_METHODS = require('../lib/http-methods');
+const checkHttpMethod = require('../lib/check-http-method');
+
+describe('checkHttpMethod function ->', () => {
+    HTTP_METHODS.forEach(method => {
+        it(`should checked as true for "${method}" method`, done => {
+            const result = checkHttpMethod(method);
+            expect(result).toBe(true);
+            done();
+        });
+    });
+
+    [null, undefined, NaN, 'wrong_method', 42].forEach(method => {
+        it(`should checked as false for incorrect "${method}" method`, done => {
```

Comment:
I was thinking about string)
It is unlikely, that sane developer is going to add `null`, `undefined` or `NaN` to http methods.

Reply:
And I propose to change spec name to 'should be able to detect an incorrect http method'.

Reply:
+